### PR TITLE
feat(RELEASE-1057): fix linting issue in check-fbc-packages

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: tasks/check-fbc-packages/check-fbc-packages.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: check-fbc-packages
   labels:
-    app.kubernetes.io/version: "0.4.0"
+    app.kubernetes.io/version: "0.5.0"
```
Comment (mmalina) [inline tasks/check-fbc-packages/check-fbc-packages.yaml]: It's more of a bugfix, so I would suggest "0.4.1" here.

**Review by mmalina — COMMENTED** [review]
Other than the minor version thing, it LGTM.

**Review by johnbieren — COMMENTED** [review]
LGTM once Martin's version comment is addressed

File: tasks/check-fbc-packages/check-fbc-packages.yaml (hunk 1 above)
Comment (jinqi7) [reply tasks/check-fbc-packages/check-fbc-packages.yaml]: Updated, thanks

**Review by mmalina — APPROVED** [review]

**Review by johnbieren — APPROVED** [review]

**Pull request merged**